spec/oga/xml/element_spec.rb

```diff
@@ -161,6 +168,14 @@
       @element.get('class').should == 'foo'
     end
 
+    it 'converts symbols to strings in names' do
```

Comment:
This is a bit picky but could you change this to `it 'supports the use of Symbols for attribute names'`?